crates/nu-command/tests/commands/network/http/put.rs

```diff
@@ -171,18 +171,25 @@ fn http_put_timeout() {
     let _mock = server
         .mock("PUT", "/")
         .with_chunked_body(|w| {
-            thread::sleep(Duration::from_secs(1));
+            thread::sleep(Duration::from_secs(10));
```

Comment:
Wondering if the `server` joins on those threads on drop or if they get ditched. But for now more reliable CI takes priority over trying to squeeze out the fastest test run (looking at the plugin timeout tests we have bigger fish to fry in that regard)